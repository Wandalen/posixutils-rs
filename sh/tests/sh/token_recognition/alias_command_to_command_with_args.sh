alias test='echo hello'

test
test world
test test

alias cmd='test world'

cmd
cmd hello
cmd cmd