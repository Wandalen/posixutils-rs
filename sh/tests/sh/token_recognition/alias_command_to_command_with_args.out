hello
hello world
hello test
hello world
hello world hello
hello world cmd
